test/scan/scanPerf.chpl

```diff
@@ -0,0 +1,32 @@
+use Time, Memory;
+
+// compute a target problem size if one is not specified
+config param memFraction = 0;
+const totalMem = here.physicalMemory(unit = MemUnits.Bytes);
+const defaultN = if memFraction == 0 then 10
+                                     else (totalMem / numBytes(int)) / memFraction;
+
+config const n = defaultN,
+             printTiming = false,
+             printArray = true;
+
+var A: [1..n] int = 1;
+
+// time scan
+const startTime = getCurrentTime();
```

Comment:
This is probably just a style nit, but I really like using the `Timer` instead of `getCurrentTime()`.

Reply:
It's so much more work though!!!!

Reply:
I usually write it as something like:

```chpl
// time scan
var t: Timer; t.start();
var B = + scan A;
t.stop();

if printTiming then
  writeln("Scan of ", n, " elements took ", t.elapsed(), " seconds");
```

which is the same number of lines. I prefer this because: 
 - it's easy to find the start/stop in generated code (lame reason)
 - `getCurrentTime()` returns time since midnight which means timings are super broken if you happen to start and stop across a day boundary

Reply:
Yeah, I always remember the midnight thing but shrug it off thinking "eh... what're the chances?"  It's less the number of lines or amount of code and more that I have to remember four things to type rather than one.